### ISSUE TITLE
Feedback

### ISSUE THREAD

**Issue by ErickNavarroD** [stat545ubc-2022/runzw_mini_data_analysis#1]
Hi! 
Nice work for this assignment. 
The only issue I found was during the first task. You were supposed to explore 4 datasets, and only worked with 2. Also, in the ones you explored, you did not show 3 or more attributes and did not use a dplyr function to do it, as it is indicated. Finally, you added no explanations on why you chose the datasets at the end. 
Other than that, your assignment was very good. Just as a comment, density plots are not the best plots to visualize discrete data, they are good for continuous data. Bar plots are better for your task.

**Comment by runzw** [comment]
Thank you for the feedback.

Can I get a detailed breakdown for my grade to show where I lost points?

**Comment by ErickNavarroD** [comment]
Yes, in canvas you can see the rubric and the grade you got in each category

**Comment by runzw** [comment]
https://github.com/stat545ubc-2022/runzw_mini_data_analysis/blob/8bc90313d166732316dcae3e7f6a9f7da4942e6f/mini_data_analysis_1.rmd#L111 Does this line count as a brief explanation of the choices?

**Comment by ErickNavarroD** [comment]
Sorry, I missed that line! I will update your grade

**Comment by runzw** [comment]
Thank you!

**Comment by runzw** [comment]
Also, can I see how many points are each of the three places here worth? It only shows the total number of points deducted.
![1666660742486](https://user-images.githubusercontent.com/78052475/197659705-c02654b1-eeb7-43be-97d8-abcc7c6bc069.jpg)

**Comment by ErickNavarroD** [comment]
I don't think you can see them, but I'm happy to provide an explanation. Each full exploration is weighted equally, which means that exploring each dataset is worth 2.25 points, and 1 point is allocated to providing an explanation of the choices 
You explored only 2 of the 4 datasets that were required, which deducts 4.5 points. Also, from the other datasets that you did explored, you did not use a dplyr function to explore the datasets, and did not show 3 or more attributes, as was required, which deducts 1.25 from each one of your explorations. I allocated one point per exploration because you did show some attributes, but it was not as many as it was required nor how you were asked to do it. Let me know if that makes sense! 👍🏽
